### PR TITLE
Generic contentlist block template

### PR DESCRIPTION
**Pull request by itkg-ngilain**

### CODE REVIEW

**Comment by nicolasThal** [comment]
:+1:

File: DisplayBundle/Resources/public/block/ContentList/css/style.css [hunk 1]
```diff
@@ -1,16 +1,18 @@
-#main .content-list {
+.content-list {
     margin-top: 30px;
 }
-#main .content-list div {
+
+.content-list div {
```
Comment (juchi) [inline DisplayBundle/Resources/public/block/ContentList/css/style.css]: not quite optimal performance-wise, isn't it possible to add some direct descendant selector ">" or specified class ?

File: DisplayBundle/Resources/public/block/ContentList/css/style.css [hunk 2]
```diff
@@ -1,16 +1,18 @@
...
     float: left;
     margin: 15px 40px 15px 0;
     border: 1px solid #1EA1B9;
-    padding: 15px;
+    padding: 5px 15px 15px 15px;
     box-sizing: border-box;
 }
-#main .content-list div h2 {
+
+.content-list div h2 a {
```
Comment (juchi) [inline DisplayBundle/Resources/public/block/ContentList/css/style.css]: idem as previous remark

File: DisplayBundle/Resources/public/block/ContentList/css/style.css [hunk 3]
```diff
@@ -18,19 +20,11 @@
...
-}
-#main .content-list div a {
-    position: absolute;
-    top: 0;
-    right: 0;
-    display: inline-block;
-    width: 100%;
-    height: 100%;
+
+.content-list div p {
```
Comment (juchi) [inline DisplayBundle/Resources/public/block/ContentList/css/style.css]: idem

File: DisplayBundle/Resources/public/block/ContentList/css/style.css [hunk 4]
```diff
@@ -18,19 +20,11 @@
...
+    margin-top:2px;
 }
-#main .content-list div:nth-child(even) {
+
+.content-list div:nth-child(even) {
```
Comment (juchi) [inline DisplayBundle/Resources/public/block/ContentList/css/style.css]: idem

File: DisplayBundle/Resources/public/block/ContentList/css/style.css (hunk 1 above)
Comment (itkg-ngilain) [inline DisplayBundle/Resources/public/block/ContentList/css/style.css]: yes it's possible :)

**Comment by itkg-ngilain** [comment]
@juchi : update done

**Comment by juchi** [comment]
:+1:

**Pull request merged**